src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#microsoft/dotnet:2.2-sdk
+FROM microsoft/dotnet@sha256:7d8256eead49252ac2de7079268659102f44a6e40e7890fec2a7633d0b374470 AS build
+
+# Copy common, used for exposing language ini files
+WORKDIR /src/AltinnCore/
+COPY src/AltinnCore/ .
+
+# Copy receipt backend
+WORKDIR /src/Altinn.Platform/
+COPY src/Altinn.Platform/Altinn.Platform.Receipt/ ./Altinn.Platform.Receipt
+
+# Build and publish
+WORKDIR /src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/
```

Comment:
You can skip the previous sted just by doing 

```
WORKDIR /src/Altinn.Patform/Altinn.PlatformReceipt/Receipt/
COPY src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/ .
```

Unless you need some files in `src/Altinn.Platform/Altinn.Platform.Receipt/`-folder

Reply:
Ah, good catch. Fixed :)